integration/kubernetes/e2e_conformance/run.sh

```diff
@@ -72,7 +72,7 @@ run_sonobuoy() {
 	[ "$CI" == true ] && cat "$e2e_result_log"
 
 	# Check for Success message on the logs.
-	grep -aq " 0 Failed" "$e2e_result_log"
+	grep -aq " 0 Failed" "$e2e_result_log" || die "A failure has been found"
```

Comment:
Nit: Is it worth referencing the log file in the error? Either specifying the name of the log or actually displaying it?

```sh
grep -aq " 0 Failed" "$e2e_result_log" || die "A failure has been found: $(cat $e2e_result_log)"
```

Reply:
Do we save that log in the CI results (Jenkins artifacts?). In which case we could just log the results of the `grep` output, rather than the whole log itself.

Reply:
@jodh-intel We have an [info](https://github.com/kata-containers/tests/blob/d2ed1c0140a281dd286143987bea408e776f02e2/integration/kubernetes/e2e_conformance/run.sh#L69) where we specify the log location and we also print the whole tests output to the console log . 

@grahamwhaley We still do not save this on the artifacts when running on Jenkins, but I will add that on the jenkins job.